lib/faker/default/beer.rb

```diff
@@ -5,34 +5,106 @@ class Beer < Base
     flexible :beer
 
     class << self
+      ##
+      # Produces a random beer name.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Beer.name #=> "Pliny The Elder"
+      #
+      # @faker.version next
       def name
         fetch('beer.name')
       end
 
+      ##
+      # Produces a random beer style.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Beer.style #=> "Wood-aged Beer"
+      #
+      # @faker.version next
       def style
         fetch('beer.style')
       end
 
+      ##
+      # Produces a random beer hops .
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Beer.hop #=> "Sterling"
+      #
+      # @faker.version next
       def hop
         fetch('beer.hop')
       end
 
+      ##
+      # Produces a random beer yeast.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Beer.yeast #=> "5335 - Lactobacillus"
+      #
+      # @faker.version next
       def yeast
         fetch('beer.yeast')
       end
 
+      ##
+      # Produces a random beer malts.
```

Comment:
I think this should be singular?

```suggestion
      # Produces a random beer malt.
```